README.md

```diff
@@ -1,2 +1,74 @@
-# rfcs
-RFCs for changes to Google Cartographer projects
+# Cartographer RFCs
+[Cartographer RFCs]: #cartographer-rfcs
+
+For changes more substantial than simple bugfixes and documentation improvements
+we ask that these be put through a very lightweight design review process.
+
+"RFC" stands for "request for comments" and is used by many projects, e.g.
+rust-lang, to establish a consistent and controlled path for new features to
+enter their project.
```

Comment:
how about adding this in the end?

", so that new contributors can anticipate short- and long-term development roadmap of the project."

Reply:
Done